src/index.html

```diff
@@ -22,6 +22,442 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="header__logo">
```

Comment:
The logo must be link
```suggestion
      <a class="header__logo">
```